README.md

```diff
@@ -2141,6 +2142,92 @@ def reply_raw() -> manual[RawReply]:
 
 [Not yet implemented.](https://github.com/demergent-labs/azle/issues/496)
 
+### Timer APIs
+
+-   [clear timer](#clear-timer)
+-   [set timer](#set-timer)
+-   [set timer interval](#set-timer-interval)
+
+#### clear timer
+
+Examples:
```

Comment:
Can you add a more information section like elsewhere in the documentation and link to the timers documentation on internetcomputer.org? If you could do this for all of the APIs that would be great